### PR TITLE
Add Testing Library

### PR DESCRIPTION
**Pull request by nickmccurdy**

### CODE REVIEW

**Comment by nickmccurdy** [comment]
Fixed the build and merge conflicts

**Comment by jamiebuilds** [comment]
Turned on some features for you:

- [Vanity Invite URL](https://support.discord.com/hc/en-us/articles/115001542132-Server-Vanity-URLs)
- [Server Banner & Invite Splash](https://support.discord.com/hc/en-us/articles/360028716472-Server-Banner-Background-Invite-Splash-Image)

Do you want to update your invite code to something like `discord.gg/testing-library`?

**Comment by nickmccurdy** [comment]
Thanks, we're using the vanity invite URL now.

File: communities.json [hunk 1]
```diff
@@ -742,6 +742,14 @@
       "quote": "We certainly think it's the most convenient platform for our communications.",
       "inviteCode": "tokio",
       "githubUrl": "https://github.com/tokio-rs"
+    },
+    {
+      "logo": "testinglibrary.png",
+      "title": "Testing Library",
+      "quote": "Simple and complete testing utilities that encourage good testing practices",
```
Comment (jamiebuilds) [inline communities.json:749]: This quote is intended for display as a comment on your Discord server, not a description of the library.

Comment (nickmccurdy) [reply communities.json]: Is it better now?

**Comment by jamiebuilds** [comment]
Thanks!

**Pull request merged**